### ISSUE TITLE
some questions about the ops of openvino and onnx

### ISSUE THREAD

**Issue by shixinlishixinli** [microsoft/onnxruntime#1527]
openvino support more than 32 ops. why ONNX Runtime only can support 32 ops in OpenVINO? if i want to add a new op in ONNX using openvino, what should i do ?

**Comment by jywu-msft** [comment]
@smkarlap @suryasidd ,  
any suggestions/guidance?

**Comment by suryasidd** [comment]
For initial release, we only added the operators that are needed to support the models in the onnx model zoo (Haven't added support for the models that were added  in onnx model zoo recently like ssd and mask rcnn). So we left out some of the ops because there were corner cases that needed to be handled for the unit tests to pass. 

We are currently adding support to the rest of the operators as well. Are you looking for any specific ops? If so, we can prioritize and add them sooner.
